spec/controllers/users/phones_controller_spec.rb

```diff
@@ -3,143 +3,6 @@
 describe Users::PhonesController do
   include Features::MailerHelper
 
-  describe '#phone' do
-    let(:user) { create(:user, :signed_up, with: { phone: '+1 (202) 555-1234' }) }
-    let(:second_user) { create(:user, :signed_up, with: { phone: '+1 (202) 555-5678' }) }
-    let(:new_phone) { '202-555-4321' }
-
-    context 'user tries to change phone settings' do
-      before do
-        stub_sign_in(user)
-
-        stub_analytics
-        allow(@analytics).to receive(:track_event)
-
-        put :update, params: {
-          edit_phone_form: { otp_delivery_preference: 'foo' },
-        }
-      end
-
-      it 'does not allow delivery preference to something other than sms or voice' do
-        expect(response).to render_template(:edit)
-      end
-    end
-
-    context 'user creates a phone with existing number' do
```

Comment:
Same as above. I don't see a dup check in add_phone_spec.rb either

Reply:
Good catch on not having a dupe check.

The functionality this controller spec is testing no longer exists, it has moved to a different part of the controller. That said, I'll try to cover that with a test.

Reply:
I am digging into this one.

The current functionality results in a super weird flow. If you add a phone number you already have, it asks you to confirm it and then redirects to the account without adding an additional number.

We should probably fix this, but I think it may be outside the scope of this PR. I'll add a test to cover the existing functionality though.